source/_patterns/02-organisms/global/site-header.mustache

```diff
@@ -1,5 +1,8 @@
-<header class="site-header clearfix" data-behaviour="SiteHeader" id="siteHeader">
+<header class="site-header clearfix" data-behaviour="SiteHeader" id="siteHeader">  
   <div class="site-header__title clearfix" role="banner">
+    <div class="skip-to-content">
```

Comment:
As skip to content is now part of the site header and not its own pattern, the BEM class naming should reflect this, so the CSS class should be `site-header__skip_to_content`.